### ISSUE TITLE
osc doesn't support rpm boolean operators

### ISSUE THREAD

**Issue by dmulder** [openSUSE/osc#521]
See here: https://rpm.org/user_doc/boolean_dependencies.html
As of rpm 4.13, rpm is able to process boolean dependencies. OSC breaks this though, because it splits the boolean dependencies into multiple lines (it appears to be trying to do some pretty formatting).
For example, this:
Requires:       (keyutils or garbage)

Is morphed by osc into this:
Requires:       (keyutils
Requires:       or
Requires:       garbage)

Which obviously causes build failures.

**Comment by dmulder** [comment]
Duplicate of #389
